accounts/urls.py

```diff
@@ -1,7 +1,8 @@
 from django.urls import path
-from .views import signup
+from .views import signup, my_profile
 from django.contrib.auth.views import LoginView
 
 urlpatterns = [
     path('signup/', signup, name='signup'),
+    path('my_profile', my_profile, name='my_profile')
```

Comment:
endpoint should always be the end with slash , and also use the "-" sign instead of the _ while defining the url patterns.
Naming should also be proper for the function as it can be profile only .

Reply:
ok, sure